content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -202,6 +200,9 @@ The following table contains feature gates for graduated or deprecated features.
 | `MountPropagation` | `false` | Alpha | 1.8 | 1.9 |
 | `MountPropagation` | `true` | Beta | 1.10 | 1.11 |
 | `MountPropagation` | `true` | GA | 1.12 | - |
+| `NodeLease` | `false` | Alpha | 1.12 | 1.13 |
+| `NodeLease` | `true` | Beta | 1.14 | 1.16 |
+| `NodeLease` | `true` | Beta | 1.17 | - |
 | `PersistentLocalVolumes` | `false` | Alpha | 1.7 | 1.9 |
```

Comment:
hello @wojtek-t .  Some initial suggestions:
- Line 205: Is the state GA?
- Line 177: the `kubelet` or the Kubelet?
- Line 182: This sentence could be reworded and possibly add more explanation, spelling too. Does this make sense?
Compared to the Node resource, the Lease object is lightweight. The Lease resource improves the performance of the node heartbeats as the cluster scales?

From the KEP:
`We will use that object to represent node heartbeat - for each Node there will be a corresponding Lease object with Name equal to Node name in a newly created dedicated namespace `

Reply:
rephrased